cns/hnsclient/hnsclient_windows.go

```diff
@@ -53,6 +53,9 @@ const (
 	// Name of the loopback adapter needed to create Host NC apipa network
 	hostNCLoopbackAdapterName = "LoopbackAdapterHostNCConnectivity"
 
+	defaultHnsGwIPAddress       = "169.254.128.2"
```

Comment:
can we please add a comment explaining why this GW is being set to .2? Please explain the HNS rehydration issue which needs this GW to be different than the loopback adapter ip